sui/sources/movescription.move

```diff
@@ -244,10 +255,33 @@ module smartinscription::movescription {
         if(start_time_ms == 0){
             start_time_ms = now_ms;
         };
-        assert!(start_time_ms >= now_ms, EInvalidStartTime); 
+        assert!(start_time_ms >= now_ms, EInvalidStartTime);
+        assert!(fee_scription.tick == string(PROTOCOL_TICK), EInvalidFeeTick);
+        assert!(fee_tick_record.tick == fee_scription.tick, EInvalidFeeTick);
+
+        let deploy_fee_amount = calculate_deploy_fee(tick, epoch_count);
+        assert!(fee_scription.amount >= deploy_fee_amount, ENotEnoughDeployFee);
+        let deploy_fee = do_split(fee_scription, deploy_fee_amount, ctx);
+        //Burn the fee and Return the acc SUI to the deployer
+        let acc_in_deploy_fee = do_burn(fee_tick_record, deploy_fee, ctx);
+        let deployer: address = tx_context::sender(ctx);
+        transfer::public_transfer(acc_in_deploy_fee, deployer); 
         do_deploy(deploy_record, tick, total_supply, start_time_ms, epoch_count, mint_fee, ctx);
     }
 
+    public entry fun deploy(
```

Comment:
It might be better to directly comment out fun deploy, since after upgrade package, the deploy fun in original package ID can not work again with new VERSION number share object.

Reply:
Remove `public function` is not compatible.